generators/client/templates/vue/src/main/webapp/app/shared/date/filters.ts.ejs

```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue';
-import { format, parseISO, formatDuration } from 'date-fns';
-import * as duration from 'duration-fns';
+import { format, parseISO } from 'date-fns';
+import * as moment from 'moment';
```

Comment:
Why do we use both libs? One lib isn't enough?

Reply:
I'm creating an issue for that, it's not related to the PR

Reply:
We previously used only date-fns in this util but to be inline with what is done for Angular, we can use moment (see my comment on the PR)

I think date-fns could be entirely replaced by moment but did not wanted to go through this process as it is not in the scope of this PR

Reply:
Thanks for the explanation @ThomasCarca :)